js/app.js

```diff
@@ -731,7 +732,7 @@ $(document).ready(function () {
         // http://www.eyecon.ro/colorpicker
         color: label.color,
         onSubmit: (hsb, hex, rgb, el) => {
-          $(el).val(hex.toUpperCase());
+          $(el).val(`#${hex.toUpperCase()}`);
```

Comment:
Could you strip off any whitespace-like characters using something like `.replace(/\s/g, '')`? In case there are leading or trailing spaces (or even spaces in between characters if a user makes a typo). Sometimes they are leading spaces/tabs in hex codes when a user copies them from some website.

Reply:
Could you provide a scenario of having white-space characters when **saving** a color from the color picker? I believe the aforementioned problem only applies when a user is inputting or copying a hex code.

Reply:
On another note, saving a color should also hide the invalid message.

Reply:
I don't think saving colors from color pickers will contain spaces. 

But I think it's good to just strip off all spaces wherever the source is